CHANGELOG.md

```diff
@@ -1 +1,11 @@
 # Changelog
+
+## 0.11.0 (TBD)
+
+### Changes
+
+- Redesigned the home frontend (#20).
```

Comment:
Should we add an entry here for the token flows re-redesign?

Also, we've started adding actual links to PR numbers. Let's do it here as well.